submissions/xT0PERx/OOP exercise/index.js

```diff
@@ -0,0 +1,93 @@
+class Mammals {
```

Comment:
Plural form in class id normally denotes a collection, while here this class instantiates a single instance of an object.